src/logtab.cpp

```diff
@@ -905,8 +917,13 @@ void LogTab::UpdateStatusBar()
                 status += ", ";
             }
             status += highlightOpt.m_value;
+            if (status.size() > 100)
+            {
+                status += "...";
+                break;
+            }
         }
-        status += "}";
+        status += "}  ";
```

Comment:
The extra white spaces is for padding, so the text doesn't stick right to the border.